tests/schemas/static/test_embedded_sensornet.py

```diff
@@ -0,0 +1,30 @@
+import pytest
```

Comment:
@gdmiron could you add that test to the action, such that it is executed every time we make a merge request? Similar to [this](https://github.com/predis-h2020/digitaltwin/tree/main/.github/workflows)

Reply:
We have the tests executed in the github [action](https://github.com/predis-h2020/metadata/blob/main/.github/workflows/tests.yml)

Reply:
We have the merge still blocked for this requested change.